lib/facil/http/websockets.c

```diff
@@ -81,8 +82,9 @@ struct buffer_s resize_ws_buffer(ws_s *owner, struct buffer_s buff) {
     free_ws_buffer(owner, buff);
     buff.data = NULL;
```

Comment:
since `tmp` is known to be `NULL`, it might be easier to avoid the `else` clause and remove the `buff.data = NULL` line (so `buff.data = tmp` always executes.

This might be meaningless, especially on a good compiler, but on a terrible compiler it could improve code cache locality on the CPU by producing less instructions and having a shorter branch.